### PR TITLE
[install-and-setup] add basic typescript content

### PR DESCRIPTION
**Pull request by sarah11918**
#### Description (required)

Adds basic TypeScript content to the new install and setup page in prep for this page being moved to reference.

Allows for some basic TS info right at the beginning of working on a new project, with link for discovering more.

### CODE REVIEW

File: src/content/docs/en/install-and-setup.mdx [hunk 1]
```diff
@@ -156,29 +156,22 @@ Explore the guides below to customize your development experience.
...
-    <LinkCard
     title="Dev Toolbar"
     description="Explore the helpful features of the dev toolbar."
     href="/en/guides/dev-toolbar/"
   />
 </CardGrid>
 
+### TypeScript in Astro
+
+Astro ships with built-in support for [TypeScript](https://www.typescriptlang.org/) which can help prevent errors at runtime by defining the shapes of objects and components in your code.
```
Comment (TheOtterlord) [inline src/content/docs/en/install-and-setup.mdx:169]: nit
```suggestion
Astro ships with built-in support for [TypeScript](https://www.typescriptlang.org/), which can help prevent errors at runtime by defining the shapes of objects and components in your code.
```

**Review by TheOtterlord — APPROVED** [review]
lgtm! one comment

**Review by at-the-vr — APPROVED** [review]
LGTM 👍, saw it live on T&D

**Comment by sarah11918** [comment]
!coauthor

**Pull request merged**